App/Models/ConfigModel.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Models;
+
+use Core\Model;
+
+/**
+ * Class ConfigModel
+ * @package App\Models
+ */
+class ConfigModel extends Model
+{
+    /**
+     * Returns all the configs orderd with their class names
+     * @return array
+     * @throws \Exception
+     */
+    public function getAllConfig():array
+    {
+        $returnData = [];
+        //Grabbing the configs class
+        $sql = 'select * from configs_class order by class';
```

Comment:
why use select * if you want only classname

Reply:
because the id is also needed. Bind in the foreach on line 35.
I could probably find a better sql query rather than querying for each class